sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManager.scala

```diff
@@ -203,7 +204,12 @@ abstract class RapidsShuffleInternalManagerBase(conf: SparkConf, isDriver: Boole
       logWarning("Rapids Shuffle Plugin is falling back to SortShuffleManager because " +
         "external shuffle is enabled")
     }
-    fallThroughDueToExternalShuffle
+    val isAdaptiveEnabled = conf.get(SQLConf.ADAPTIVE_EXECUTION_ENABLED.key, "false").toBoolean
+    if (isAdaptiveEnabled) {
+      logWarning("Rapids Shuffle Plugin is falling back to SortShuffleManager because " +
```

Comment:
nit: this could be a single warning:

```
logWarning(s"Rapids Shuffle Plugin is falling back to SortShuffleManager because:\n${reasons.mkString("\n")}")
```
where `reasons` would be a `Seq` that could include `External Shuffle is enabled` and `Adaptive Query Execution is enabled`

Reply:
Good idea. I implemented that.

Reply:
Looks good!